pkg/apis/apps/v1alpha1/yurtappdaemon_types.go

```diff
@@ -73,6 +73,8 @@ type YurtAppDaemonStatus struct {
 	// +optional
 	Conditions []YurtAppDaemonCondition `json:"conditions,omitempty"`
 
+	OverriderRef string `json:"overriderRef"`
+
```

Comment:
why don't add `workloadSummaries` for yurtappdaemon?

Reply:
At the moment there wasn't any replicas related information in the yurtappdaemon status. Is it necessary to add workloadSummary into status?

Reply:
because yurtappoverrider can configure replicas of yurtappdaemon for each nodepool, so i think it is a good idea to add workloadSummaries for yurtappdaemon.